qa/suites/orch/cephadm/thrash/2-thrash.yaml

```diff
@@ -3,6 +3,13 @@ overrides:
     log-ignorelist:
     - but it is still running
     - objects unfound and apparently lost
+    - \(MON_DOWN\)
+    - \(OSDMAP_FLAGS\)
+    - flag\(s\) set
+    - \(CACHE_POOL_NO_HIT_SET\)
+    - \(PG_
+    - \(OSD_
+    - mons down:
```

Comment:
So this causes https://tracker.ceph.com/issues/64452

I don't think we need a colon behind `mons down`